src/frontend/src/btc/components/send/BtcSendForm.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import BtcSendAmount from './BtcSendAmount.svelte';
+	import BtcSendDestination from '$btc/components/send/BtcSendDestination.svelte';
+	import type { BtcAmountAssertionError } from '$btc/types/btc-send';
+	import SendForm from '$lib/components/send/SendForm.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { NetworkId } from '$lib/types/network';
+	export let networkId: NetworkId | undefined = undefined;
+	export let amount: number | undefined = undefined;
+	export let destination = '';
+	export let source: string;
+
+	let amountError: BtcAmountAssertionError | undefined;
+	let invalidDestination: boolean;
+
+	const dispatch = createEventDispatcher();
+	const close = () => dispatch('icClose');
+</script>
+
+<SendForm {source}>
+	<BtcSendDestination
+		slot="destination"
+		bind:destination
+		bind:invalidDestination
+		{networkId}
+		on:icQRCodeScan
+	/>
+
+	<BtcSendAmount slot="amount" bind:amount bind:amountError />
```

Comment:
I don't know the details, but will you add an `insufficientFunds` prop here at some point? or the check will be done differently?

Reply:
Yeah, the plan is to add `insufficientFunds` when I start working on the related part.